dockerfiles/base/Dockerfile

```diff
@@ -5,9 +5,9 @@ FROM quay.io/mlrun/python:${MLRUN_PYTHON_VERSION}-slim
 RUN apt-get update && \
     apt-get -s dist-upgrade | \
     grep "^Inst" | \
-    grep -i securi | \
+    grep -i "stable\|securi" | \
```

Comment:
I guess we should "explore" (and leave in the PR description / comment on code) what are the possible types we're filtering out by this grep, and what is the meaning of filtering in the `stable` ones (security is pretty clear, stable is not)

Reply:
This is mainly used to upgrade all installed packages, nothing really special about it, it just that I found that stable is the common for most packages that we missed